utilities/cli/tests/test_container.py

```
@@ -160,6 +161,101 @@ def test_run(
+        with (
+            patch("utilities.cli.container.Path.is_dir", return_value=True),
+            patch(
+                "utilities.cli.container.HoloHubContainer._peer_creds_sufficient",
+                return_value=True,
+            ),
+            patch("utilities.cli.container.run_command") as mock_run,
+        ):
+            options = self.container.get_display_options(True, False)
+
+        self.assertIn("DISPLAY", options)
+        self.assertIn("/tmp/.X11-unix:/tmp/.X11-unix:ro", options)
```

Ruff (0.15.12)

[error] Probable insecure usage of temporary file or directory: "/tmp/.X11-unix:/tmp/.X11-unix:ro"

(S108)